### PR TITLE
Us9 chamalie

### PR DESCRIPTION
**Pull request by Chamalie-UOM**

### CODE REVIEW

File: prime_fn.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def isPrime(x):
+    for i in range (2,x/2):
+        if (x%i==0):
```
Comment (Adeeshaj) [inline prime_fn.py:3]: is this correct?

File: prime_fn.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+def isPrime(x):
```
Comment (Adeeshaj) [inline prime_fn.py:1]: this is duplicated

**Review by Adeeshaj — CHANGES_REQUESTED** [review]

File: prime_fn.py (hunk 2 above)
Comment (Chamalie-UOM) [reply prime_fn.py:1]: file 2 is a duplicate

File: prime_fn.py (hunk 1 above)
Comment (Chamalie-UOM) [reply prime_fn.py:3]: yes

**Review by Adeeshaj — APPROVED** [review]

**Pull request merged**